src/preferences.yaml

```diff
@@ -3,4 +3,7 @@ user_preferences:
   travel-time-from-home-station-to-university-via-train: 40
   home-train-station-name: "Ludwigsburg"
   rapla-url: "https://rapla.dhbw.de/rapla/internal_calendar?key=YFQc7NlGleuSdybxizoa8NHjLLNjd9D6tjBdAvDwwzXobLEfUIsCXHwYu-Ma7QfggMDkLLj1CsQ-kB7hFJSGjcrCTGDFdUXDv_NNHoHvSJP_dG36z7u1r9aZLrggiD92Gryjvwt1kpad5g93Dkdn0A&salt=1046252309&day=3&month=10&year=2023&goto=Datum+anzeigen&pages=12#6"
-  news-interests: "Umwelt;Bitcoin"
\ No newline at end of file
+  news-interests: "Umwelt;Bitcoin"
+  weather-api-key: ""
```

Comment:
Hier kommen nur User spezifische Daten rein. Also keine API Keys außer die sind auf den User spezifisch, was sie in diesen Fällen nicht sind